sdk/compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -1615,6 +1615,33 @@ tests TestArgs{..} =
             ]
         ]
 
+    , simpleImportTest "Using explicit exports"
```

Comment:
Should we have another test case where things that are not exported but are used in the client module lead to compilation errors instead of relying on "lack of ambiguous X" errors not showing up? Or is it too complex with the current test framework?

Reply:
I'll take a look

Reply:
Everything in the suite so far asserts success, i in fact got the idea of "no error proves X" from another existing test.
Given time constraints, I don't really want to refactor this test suite :sweat_smile: